readthedocs/doc_builder/director.py

```diff
@@ -197,17 +206,31 @@ def build(self):
 
     # VCS checkout
     def checkout(self):
-        log.info(
-            "Clonning repository.",
-        )
+        """Checkout Git repo and load build config file."""
+
+        log.info("Cloning repository.")
         self.vcs_repository.update()
 
         identifier = self.data.build_commit or self.data.version.identifier
         log.info("Checking out.", identifier=identifier)
         self.vcs_repository.checkout(identifier)
 
-        self.data.config = load_yaml_config(version=self.data.version)
+        # The director is responsible for understanding which config file to use for a build.
+        # Reproducibility: Build already has a config_file set.
```

Comment:
Not sure what this comment means?

Reply:
I rewrote most of this as a comment about future work: (re)using a `build_config_file` defined for a build instead of per-project or per-version (TBD). It should be clearer now :+1: